ffi/capi/src/casemap.rs

```diff
@@ -300,8 +301,12 @@ impl From<ffi::TitlecaseOptionsV1> for TitlecaseOptions {
     fn from(other: ffi::TitlecaseOptionsV1) -> Self {
         let mut ret = Self::default();
 
-        ret.leading_adjustment = other.leading_adjustment.into();
-        ret.trailing_case = other.trailing_case.into();
+        if let Some(l) = other.leading_adjustment.into_converted_option() {
+            ret.leading_adjustment = l;
+        }
```

Comment:
nit: use `unwrap_or` like in the other options conversions
```suggestion
        ret.leading_adjustment = other.leading_adjustment.into_converted_option().unwrap_or(ret.leading_adjustment);
```